index.d.ts

```diff
@@ -1,122 +1,124 @@
-import {KEYWORD} from 'color-convert/conversions';
+import * as cssColors from 'color-name';
```

Comment:
Why did you change to this dependency? I would prefer not to add more dependencies here, even if it's just for TS.

Reply:
Because `color‑name` is used by `color‑convert`.

Reply:
Alright